libs/net/dal/Services/AVOverviewInstanceService.cs

```diff
@@ -89,19 +89,23 @@ public override AVOverviewInstance Update(AVOverviewInstance entity)
             else
             {
                 var originalItems = this.Context.AVOverviewSectionItems.Where(s => s.SectionId == section.Id).ToArray();
-                originalItems.Except(section.Items).ForEach(s =>
+                var deletedItems = originalItems.Where(x => !section.Items.Any(y => y.Id == x.Id)).ToArray();
```

Comment:
Same comment as above.  Please follow the convention.

Reply:
The initial change here is for avoiding the unnecessary delete action for some reason even though just for editing data only, however, the change is unnecessary and it is now removed, thanks